include/volk/volk_common.h

```diff
@@ -76,8 +76,9 @@
 ////////////////////////////////////////////////////////////////////////
 // C-linkage declaration macros
 // FIXME: due to the usage of complex.h, require gcc for c-linkage
+// Hope and p*** that extern "C" works for all relevant compilers nowadays.
```

Comment:
remove `and p***` please ... "Hope" is sufficient ;)

Reply:
Well. It didn't help. but yeah.